.circleci/config.base.yml

```
@@ -912,4 +960,4 @@ workflows:
           filters:
             branches:
               only:
-                - release
+                - release
```

nit: newline at end of file

```suggestion
                - release

```